### PR TITLE
ENH: Update remote modules for ITKv5.1rc02

### PR DESCRIPTION
**Pull request by hjmjohnson**
## PR Checklist
- [X] No [API changes]  (https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [X] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)
- [X] Updated API documentation (or API not changed)

### CODE REVIEW

**Comment by hjmjohnson** [comment]
Circle CI failure is not related to these changes.

File: Modules/Remote/BioCell.remote.cmake [hunk 1]
```diff
@@ -1,8 +1,50 @@
...
+#--
+#-- ### Please document here any justification for the criteria above
+#       Code style enforced by clang-format on 2020-02-19, and clang-tidy modernizations completed
+
 itk_fetch_module(BioCell
 "This module contains classes related to segmentation of biological cells.
 It has classes to represent cells' shape, color, and growth state.
 It also has classes to represent a cell genome,
 whose expression is modeled by differential equations."
+  MODULE_COMPLIANCE_LEVEL 1
```
Comment (dzenanz) [inline Modules/Remote/BioCell.remote.cmake]: Worksheet indicates this should be 2.

File: Modules/Remote/Montage.remote.cmake [hunk 2]
```diff
@@ -1,6 +1,48 @@
...
+#-- ## Compliance Level 0 star ( Code/Feature of known poor-quality or deprecated status )
+#--   - [ ] Code reviewed and explicitly identified as not recommended for use
+#--
+#-- ### Please document here any justification for the criteria above
+#       Code style enforced by clang-format on 2020-02-19, and clang-tidy modernizations completed
+
 # Contact: Dženan Zukić <dzenan.zukic@kitware.com>
 itk_fetch_module(Montage
 "Reconstruction of 3D volumetric dataset from a collection of 2D slices"
+  MODULE_COMPLIANCE_LEVEL 1
```
Comment (dzenanz) [inline Modules/Remote/Montage.remote.cmake]: Worksheed indicates level 2.

File: Modules/Remote/TextureFeatures.remote.cmake [hunk 3]
```diff
@@ -12,6 +12,7 @@ For more information, see:
   http://hdl.handle.net/10380/3574
   http://insight-journal.org/browse/publication/985
 "
+  MODULE_COMPLIANCE_LEVEL 1
```
Comment (dzenanz) [inline Modules/Remote/TextureFeatures.remote.cmake]: Why no worksheet for this module?

File: Modules/Remote/TextureFeatures.remote.cmake [hunk 4]
```diff
@@ -1,3 +1,46 @@
+#-- # Grading Level Criteria Report
+#-- EVALUATION DATE: 2020-03-01
+#-- EVALUATORS: [<<NO EVALUATOR>>,<<NO EVALUATOR>>]
+#--
+#-- ## Compliance level 5 star (AKA ITK main modules, or remote modules that could become core modules)
```
Comment (dzenanz) [inline Modules/Remote/TextureFeatures.remote.cmake]: Ah, worksheet added now :)

File: Modules/Remote/AnalyzeObjectLabelMap.remote.cmake [hunk 5]
```diff
@@ -1,4 +1,6 @@
-#-- # Grading Level Criteria
+#-- # Grading Level Criteria Report
```
Comment (dzenanz) [inline Modules/Remote/AnalyzeObjectLabelMap.remote.cmake]: This commit should be squashed with the previous one.

**Pull request merged**